packages/xo-server/src/xapi/index.mjs

```diff
@@ -192,20 +194,58 @@ export default class Xapi extends XapiBase {
         return network.$ref
       }
     })(pool.other_config['xo:migrationNetwork'])
+
+    // host ref
+    // migration network: optional and might not be supported
+    // batch size: optional and might not be supported
+    const params = [hostRef, migrationNetworkRef ?? Ref.EMPTY, this._vmEvacuationConcurrency]
```

Comment:
There is a version of XAPI which accept two params but not `Ref.EMPTY` for the second one, I'm not sure what the error will  be though.

Reply:
2 params including `hostRef`? That shouldn't happen thanks to `popParamsAndTrim` which removes any empty trailing param.